### ISSUE TITLE
Reproducible results in TarsClassifier after training with very less data(Few Shot)

### ISSUE THREAD

**Issue by smatrix017** [flairNLP/flair#2206]
Hi, I am facing an issue with TARSClassifier, where on training tars-base model with few shot examples multiple times, I am getting different results during inference each time. This issue is happening even if the training data is same and the seed values( for numpy, torch and flair) are being used. 

e.g.: 
Iteration 1: tars-base Model trained using Corpus C, getting some results R1.
Iteration 2: tars-base Model trained using Corpus C, getting some results R2.
